scene/main/node.h

```diff
@@ -118,7 +118,7 @@ class Node : public Object {
 		INTERNAL_MODE_BACK,
 	};
 
-	enum AutoTranslateMode {
+	enum AutoTranslateMode : unsigned int {
```

Comment:
Let's try to avoid platform-specific types:

```suggestion
	enum AutoTranslateMode : uint8_t {
```

Reply:
Used it for consistency with the rest of code - see https://github.com/godotengine/godot/pull/61757, in this case only relevant part is `unsigned`, not size.

Reply:
`int` to `unsigned int` is fine, and less risky for existing APIs than using another type altogether.
In general though it would be good to choose small data types; `uint8_t` should be plenty for most, this one included.
Cc @lawnjelly, maybe you got another take on this?